packages/kbn-cli-dev-mode/src/get_server_watch_paths.ts

```diff
@@ -65,6 +65,7 @@ export function getServerWatchPaths({ pluginPaths, pluginScanDirs }: Options) {
     fromRoot('x-pack/plugins/lists/server/scripts'),
     fromRoot('x-pack/plugins/security_solution/scripts'),
     fromRoot('x-pack/plugins/security_solution/server/lib/detection_engine/scripts'),
+    fromRoot('x-pack/plugins/metrics_summary/server/scripts'),
```

Comment:
Should this be `metrics_entities`?

```suggestion
    fromRoot('x-pack/plugins/metrics_entities/server/scripts'),
```